plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/util.ts

```diff
@@ -41,13 +41,35 @@ export const parseRepoUrl = (repoUrl: string) => {
     );
   }
   const host = parsed.host;
+  const type = parsed.searchParams.get('type');
```

Comment:
No need to pass the type explicitly afaik, it should be possible to deduce from the host using the scmIntegrations.